jakartaee/10/coreprofile/22.0.0.10-beta-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,157 @@
+[javatest.batch] Number of Tests Passed      = 1
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+[javatest.batch] ********************************************************************************
+[javatest.batch] PASSED........com/sun/ts/tests/signaturetest/caj/CAJSigTest.java#signatureTest
+
+
+Stage Name: CDI TCK
+[INFO] Tests run: 725, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,017.849 s - in TestSuite
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 725, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.881 s - in org.jboss.weld.langmodel.tck.LangModelTckTest
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO] /jakarta/conf/cdi-tck/target/surefire-reports/sigtest/TEST-liberty-cdi-tck-runner-4.0.5.xml: 0 failures in /jakarta/conf/cdi-tck/target/api-signature/cdi-api-jdk11.sig
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 29.188 s
+
+
+Stage Name: Core TCK
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 13, Failures: 0, Errors: 0, Skipped: 0
+[INFO] 
+[INFO] 
+[INFO] --- maven-failsafe-plugin:3.0.0-M6:verify (verify) @ core-profile-tck-runner ---
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 01:24 min
+
+
+Stage Name: Dependency Injection TCK
+[INFO] Tests run: 50, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.466 s - in weld.SampleBootstrapTCK
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 50, Failures: 0, Errors: 0, Skipped: 0
+
+
+Stage Name: JSON Binding TCK
+[INFO] Results:
+[INFO] 
+[WARNING] Tests run: 295, Failures: 0, Errors: 0, Skipped: 5
+
+
+Stage Name: JSON Processing TCK
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 179, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO] Tests run: 18, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.08 s - in ee.jakarta.tck.jsonp.pluggability.jsonprovidertests.ClientTests
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 18, Failures: 0, Errors: 0, Skipped: 0
+
+
+Stage Name: RESTful Web Services TCK
+[INFO] Results:
+[INFO] 
+[ERROR] Failures: 
+[ERROR]   ClientExceptionsIT.shouldThrowMostSpecificWebApplicationException:93 
+Expected: jakarta.ws.rs.RedirectionException but: was jakarta.ws.rs.InternalServerErrorException
+     but: was <ee.jakarta.tck.ws.rs.spec.client.exceptions.ClientExceptionsIT$$Lambda$624/0x000000002ddb9710@eb1840e6>
+[ERROR] Errors: 
+[ERROR]   JsonbContextProviderIT.shouldUseApplicationProvidedJsonbInstance:106 » InternalServerError HTTP 500 Internal Server Error
+[INFO] 
+[ERROR] Tests run: 2647, Failures: 1, Errors: 1, Skipped: 59
+----
```

Comment:
Is this fine?

Reply:
When we submit the CCR, just add a comment to mention the challenge number.

Reply:
That is why the challenges are stated above.

Reply:
It's not ideal, but it's the approach that was discussed/approved on the platform call today.  I'm told that we cannot wait for an updated version of the TCK.